src/main/java/com/shs/b2bm/claim/service/utils/ExtractValueFromJson.java

```diff
@@ -0,0 +1,59 @@
+package com.shs.b2bm.claim.service.utils;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ExtractValueFromJson {
```

Comment:
```suggestion
public class ValidationConfig {
```
This does more than just extracting values